src/pages/MovieDetailsPage.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <div class="movie">
+    <BreadCrumbs
+      :steps="steps"
+      :back-route="{ name: 'Movies' }"
+      class="mb-64"
+    />
+    <div class="movie__details mx-sm-24 mb-64">
+      <div class="movie__details-wrapper">
+        <SectionTitle class="mb-32">{{ movie?.title }}</SectionTitle>
+        <div class="movie__details-basics">
+          <CustomChip class="mr-16">{{ movie?.genre.name }}</CustomChip>
+          <span class="movie__details-text">{{ releaseYear }}</span>
+          <EllipseIcon class="dot-divider" />
+          <span class="movie__details-text">{{ movie?.length }}</span>
+        </div>
+        <SectionSubtitle class="movie__details-description">{{
+          movie?.description
+        }}</SectionSubtitle>
+      </div>
+      <img class="movie__details-image" :src="movie?.poster_url" />
```

Comment:
I would like the img to fill the height of the content in movie__details-wrapper, I failed. I'm open to suggestions.